app/controllers/trains_controller.rb

```diff
@@ -3,10 +3,24 @@ def index
     @trains = TrainApi.new.index
   end
 
+  def edit
+    flash["errors"] = nil
+    @train = TrainApi.new.show(params)
```

Comment:
imo we should pass JUST the id, since thats all we need for this

Reply:
should we be using a helper method for this like all the other controllers? instead of setting up the instance variable

Reply:
> should we be using a helper method for this like all the other controllers? instead of setting up the instance variable

i tried using a helper for `@trains` but it wont work with rendering and updating the form

Reply:
did you try like all the other controllers in the app? since they dont look like they are doing something you couldnt
the exact same way should be fine, so that we arent using `@train` in the view files.